README.md

```diff
@@ -21,7 +36,15 @@ compatibility. When possible, upgrade your Conjur version to match the
 when using integrations, choose the latest suite release that matches your Conjur version. For any 
 questions, please contact us on [Discourse](https://discuss.cyberarkcommons.org/c/conjur/5).
 
-### Binaries (Recommended)
+## Prerequisites
+
+The following are prerequisites to using the Terraform Provider Plugin.
+
+### 1.Conjur setup
+
+You need to Setup Conjur OSS locally. To setup Conjur OSS follow Conjur quickstart [Cyberark-Conjur-OSS](https://github.com/cyberark/conjur-quickstart)
```

Comment:
@itsbrugu What about Enterprise customers? Is it safe to point customers to the official deployment documentation and say that you must have Conjur configured?

Reply:
@gcandiloro Please follow https://github.com/cyberark/terraform-provider-conjur/blob/1294ddbc2adb3e78a20d0832372c3e112c51a15b/README.md for latest updates